build.sbt

```diff
@@ -71,6 +71,7 @@ testOnlyJVM := {
   val runBenchmarkTests = test.in(benchmarks, Test).value
   val runContribTests = test.in(contribJVM, Test).value
   val runTests = test.in(testsJVM, Test).value
+  val propertyTests = compile.in(testkit, Test).value
```

Comment:
Just checking that these compile, as long as they compile we should be able to use git bisect to track down regressions in case something goes wrong.